scripts/linters/css_linter.py

```diff
@@ -58,6 +59,28 @@ def all_filepaths(self):
         """Return all filepaths."""
         return self.files_to_lint
 
+    def _get_trimmed_summary_message(self, lint_messages):
```

Comment:
Because it is a method in a class.

Reply:
but it doesn't need the self, right? can this be a static method for the class? or should we move the lint messages as a class property?

Reply:
This is not a static method, we are changing things dynamically inside it then why should we change it to static method and it is not a class property either.

Reply:
>  What will be the advantage of static method or property over this?

1. It will indicate that this message doesn't change any internal property of the object.
2. We will avoid adding unwanted params.

Let me know what you think, I'm fine with either!

Reply:
I think static method is fine.